transaction-flow.md

```diff
@@ -439,12 +439,13 @@ Fidelity Bank: 240100
 ```
 
 ##### UGX::Mobile
+Please note that UGX::Mobile payouts are currently in beta phase.
 
 ```javascript
 "details": {
   "first_name": "First",
   "last_name": "Last",
-  "phone_number": "414123456"
+  "phone_number": "256772123456"
     // local Ugandan format
```

Comment:
In the [API Specification project](https://github.com/bitpesa/bitpesa-api-specification/pull/159/files#diff-425157696e8824be922c0eb3145a2e11R319), this is listed as `# local or international Ugandan format` - we should make sure both are consistent.

Reply:
Done.